File: db/event_type_localized.php
<?php

require_once("entity.php");

/**
 * Class for accessing, manipulating, creating and deleting records in event_type_localized.
*/

class Event_Type_Localized extends Entity
{

  public function __construct($select = array())
  {
    $this->table = "event_type_localized";
    $this->order = "";
    $this->domain = "localization";
    $this->field['event_type_id']['type'] = 'INTEGER';
    $this->field['event_type_id']['not_null'] = true;
    $this->field['language_id']['type'] = 'INTEGER';
    $this->field['language_id']['not_null'] = true;
    $this->field['name']['type'] = 'VARCHAR';
    $this->field['name']['length'] = 64;
    $this->field['name']['not_null'] = true;
    $this->field['event_type_id']['primary_key'] = true;
    $this->field['language_id']['primary_key'] = true;
    parent::__construct($select);
  }

}

?>

File: pentabarf/lib/momomoto/tables/event_type_localized.rb
module Momomoto
  class Event_type_localized < Base
    def initialize
      super
      @domain = 'localization'
      @fields = {
        :event_type_id => Datatype::Integer.new( {:not_null=>true, :primary_key=>true} ),
        :language_id => Datatype::Integer.new( {:not_null=>true, :primary_key=>true} ),
        :name => Datatype::Varchar.new( {:length=>64, :not_null=>true} )
      }
    end
  end
end
